src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/StocksTransformerImpl.java

```diff
@@ -1,6 +1,8 @@
 package uk.gov.companieshouse.web.accounts.transformer.smallfull.impl;
```

Comment:
Can we format this file to prevent some lines going over the required 100 character line length.

Reply:
?

Reply:
done